xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-send/xwiki-platform-mail-send-default/src/main/java/org/xwiki/mail/script/ScriptMimeMessage.java

```diff
@@ -86,7 +86,7 @@
      * @return the Mime Body Part object that was added. Returning it allows script to make modifications to that body
      *         part after it's been set (get/set some headers, etc)
      * @param mimeType the mime type of the content parameter
-     * @param content the content to include in the mail
+     * @param content the content to include in the mail, also directly accepts BodyPart objects
```

Comment:
I've just improved the javadoc for the content parameter. Could you please update your javadoc too? thanks!

Reply:
where did you update it?

Reply:
ah I found it